### DISCUSSION TITLE
Does easyOCR supports hocr format?

### DISCUSSION THREAD

**Discussion by himasai9711** [JaidedAI/EasyOCR#1090] [Q&A]
I was working on table data extraction from pdf files I wanted to know does easyocr emits in hocr format?, if so can you guide me how to do like method or give me proper documentation.

**Comment by deajan** [comment]
Anyone ? I'm working on bringing EasyOCR to paperless-ngx, and this is a major stopper for me.

**Comment by thiswillbeyourgithub** [comment]
I made [OCR_with_format](https://github.com/thiswillbeyourgithub/OCR_with_format) and would like to use easyOCR instead of tesseract but it would be insanely easier if I could use hocr as a common format
